### ISSUE TITLE
Feature Request: Add a GUI option for windows build #

### ISSUE THREAD

**Issue by t-jonesy** [IsaacMarovitz/Whisky#18]
A GUI option to set the Windows build number would probably help those who can't make the registry mod themselves:
```
reg add 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion' /v CurrentBuild /t REG_SZ /d 19042 /f
reg add 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion' /v CurrentBuildNumber /t REG_SZ /d 19042 /f
```
Those registry changes were required to get Diablo IV running

**Comment by Frityet** [comment]
Im making a registry editor (#13) so i guess I could include this

**Comment by Moeinich** [comment]
> A GUI option to set the Windows build number would probably help those who can't make the registry mod themselves:
> 
> ```
> reg add 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion' /v CurrentBuild /t REG_SZ /d 19042 /f
> reg add 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion' /v CurrentBuildNumber /t REG_SZ /d 19042 /f
> ```
> 
> Those registry changes were required to get Diablo IV running

how do you manually edit the registry keys with Whisky?

**Comment by Daerkle** [comment]
I managed it through the cumbersome way that I navigated to the regedit.exe (with whisky) in the Bottle and then executed it

**Comment by Moeinich** [comment]
> I managed it through the cumbersome way that I navigated to the regedit.exe (with whisky) in the Bottle and then executed it

Yeah I figured out its easier to actually just go to the wine directory from the mac and open the regedit config file and then just edit the two values from there and save :)

**Comment by IsaacMarovitz** [comment]
Added, coming in next update
